src/sentry/static/sentry/app/api.jsx

```diff
@@ -193,11 +193,11 @@ export class Client {
 
   requestPromise(path, options = {}) {
     return new Promise((resolve, reject) => {
+      const {isjqXHR} = options;
```

Comment:
Can we name this something more generic like `includeAllArgs` and destructure this in the function parameters? This way it doesn't get unnecessarily get passed to `this.request`:

`requestPromise(path, {includeAllArgs, ...options} = {}) {`